### PR TITLE
PR from Geoffrey without conflicts

### PR DESCRIPTION
**Pull request by moritzpschwarz**
This branch continues the work that was started on #52 but without merge conflicts. 

#52 will be closed.

### CODE REVIEW

File: tests/testthat/.DS_Store
Comment (moritzpschwarz) [inline tests/testthat/.DS_Store:1]: remove

File: tests/testthat/Rplots.pdf
Comment (moritzpschwarz) [inline tests/testthat/Rplots.pdf:1]: remove

File: tests/testthat/_snaps/2-forecastmodel/Forecast_plot.new.png
Comment (moritzpschwarz) [inline tests/testthat/_snaps/2-forecastmodel/Forecast_plot.new.png:1]: check

File: tests/testthat/testdata/.DS_Store
Comment (moritzpschwarz) [inline tests/testthat/testdata/.DS_Store:1]: remove

**Review by moritzpschwarz — COMMENTED** [review]
A few small comments that I will look at this week

File: tests/testthat/testdata/.DS_Store
Comment (Geoffrey-Harper) [reply tests/testthat/testdata/.DS_Store:1]: I can now see comments! I will remove these files tomorrow evening when I have time

**Pull request merged**